test/LoadTest/cache_refresh/README.md

```diff
@@ -69,25 +69,34 @@ These tests are used to verify the cache refresh in a balanced system.
   Parameters:
   ```
      <url>: host and port used by requests_listener.py script (MANDATORY)
+     -reset: is used to reset the listener previously (Optional)
```

Comment:
I'm not sure about the meaning of "reset the listener previously".

Do you mean "reset the listener before starting/initiating the test"?